docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3'
+
+services:
+  proxy:
+    image: nginx:stable
+    links:
+      - app
+    ports:
+      - "8082:8080"
+    volumes:
+      - ./build/proxy/nginx.conf.template:/etc/nginx/conf.d/nginx.conf.template:ro
+      - ./build/proxy/includes:/etc/nginx/conf.d/includes:ro
+      - ./web:/usr/share/nginx/www/spenden.wikimedia.de/current/web:ro
+    environment:
+      - NGINX_HOST=spenden.wikimedia.de
+      - NGINX_PORT=8080
+    command: /bin/bash -c "envsubst '$$NGINX_HOST $$NGINX_PORT' < /etc/nginx/conf.d/nginx.conf.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
+
+  app:
+    build: ./build/app
+    expose:
+      - "9000"
+    volumes:
+      - .:/usr/share/nginx/www/spenden.wikimedia.de/current
+      - var-data:/usr/share/nginx/www/spenden.wikimedia.de/current/var
+    working_dir: /usr/share/nginx/www/spenden.wikimedia.de/current
+    links:
+      - database
+      - mailhog
+
+  database:
+    image: mysql:5.6
+    environment:
+      - MYSQL_RANDOM_ROOT_PASSWORD=yes
+      - MYSQL_USER=fundraising
+      - MYSQL_PASSWORD=INSECURE PASSWORD
+      - MYSQL_DATABASE=fundraising
+    expose:
```

Comment:
@wiese Is there a reason why there is no volume defined for the database? The current setup will delete all data and tables as soon as docker-compose is stopped.

Reply:
No explicit reason but I did not consider it needed so far. The container (and its volume) will be reused unless you explicitly remove it.
Mind "Where to Store Data" in https://hub.docker.com/_/mysql/